backend/Dockerfile

```diff
@@ -45,15 +45,19 @@ FROM debian:bookworm-slim AS debug
 WORKDIR /app
 
 COPY --chmod=777 --from=build-debug /app/target/debug/arcadia-index .
-CMD ["arcadia-index"]
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+      apt-get install -y openssl 
+CMD ["./arcadia-index"]
 
 FROM debian:bookworm-slim
 WORKDIR /app
 
 COPY --chmod=777 --from=build-prod /app/target/release/arcadia-index .
 
-# COPY docker-entrypoint.sh /usr/local/bin/
-# RUN chmod +x /usr/local/bin/docker-entrypoint.sh
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+      apt-get install -y openssl 
```

Comment:
@shahoob does this look right or is there a better way to make openssl available to the binary at runtime?

Reply:
ah was hoping I could get away with using the build cache but I didn't realize the base images where different, thanks!

Reply:
resolved by [638bf69](https://github.com/Arcadia-Solutions/arcadia/pull/56/commits/638bf693c25b19c06d09b7d743cb839db2048435)